packages/rancher-logging/overlay/templates/loggings/rke/logging-rke.yaml

```diff
@@ -14,10 +14,11 @@ spec:
       tag: {{ .Values.images.fluentbit.tag }}
     inputTail:
       Tag: "rke"
-      Path: "/rke/*.log"
```

Comment:
What did this path previously do ?

Reply:
previously this was how we collected the rke logs but it yielded no meta data, this pr replaced it.